inference_rules.adoc

```diff
@@ -515,9 +515,8 @@ much much smaller than the non-zero weights it produces.
 Calibration is allowed and must only use the calibration data set provided by
 the benchmark owner. Submitters may choose to use only a subset of the calibration data set.
 
-Additionally, for image classification using MobileNets-v1 224 and object
-detection using SSD-MobileNets-v1, MLPerf will provide a retrained INT8
-(asymmetric for TFLite and symmetric for pyTorch/ONNX) model. Model weights and
+Additionally, MLPerf may provide a retrained INT8 model when post-training quantization cannot reach
+the desired accuracies in with some models. Model weights and
```

Comment:
I suggest just "Additionally, MLPerf may provide an INT8 reference for some models. Model weights and <...>"

Because whether accuracy can be achieved with PTQ is a submitter-specific criterion.